springfield/base/templates/base-protocol.html

```diff
@@ -28,7 +28,7 @@
     {% block extra_meta %}{% endblock %}
 
     {% block shared_meta %}
-    <title>{% filter striptags %}{% block page_title_full %}{% block page_title_prefix %}{% endblock %}{% block page_title %}{% endblock %}{% endblock page_title_full %}{% block page_title_suffix %} — Firefox{% endblock %}{% endfilter %}</title>
+    <title>{% filter striptags %}{% block page_title_full %}{% block page_title_prefix %}{% endblock %}{% block page_title %}{% endblock %}{% endblock page_title_full %}{% block page_title_suffix %} — Firefox.com{% endblock %}{% block page_title_region %}{% endblock %}{% endfilter %}</title>
```

Comment:
Given this is a global base template, I'd ask around first if that is a pattern worth encouraging in the first place.

My understanding is when a customization is necessary, there's a reason, and it should be explicit at that exact page, with ideally tailored titles fully.

(Not sure if the concern is still valid after these years though.)